src/Storage/Storage/Blob/Cmdlet/CopyAzureStorageBlob.cs

```diff
@@ -99,9 +99,9 @@ public string SrcContainer
         [ValidateSet("Block", "Page", "Append", IgnoreCase = true)]
         public string DestBlobType { get; set; }
 
-        [Parameter(HelpMessage = "Block Blob Tier, valid values are Hot/Cool/Archive. See detail in https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers", Mandatory = false)]
+        [Parameter(HelpMessage = "Block Blob Tier, valid values are Hot/Cool/Archive/Cold. See detail in https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers", Mandatory = false)]
```

Comment:
```suggestion
        [Parameter(HelpMessage = "Block Blob Tier, valid values are Hot/Cool/Archive/Cold. See detail in https://docs.microsoft.com/azure/storage/blobs/storage-blob-storage-tiers", Mandatory = false)]
```
Remove culture localization information from url if possbile